packages/react/src/utils/useOutboundClickListener.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { MatomoInstance } from '../types'
+
+const useOutboundClickListener = ({ trackLink }: MatomoInstance) => {
+  const handleOutboundClick = (event: MouseEvent) => {
+    if (!event.target) {
```

Comment:
perhaps make it `if (!event.target && typeof window !== 'undefined') {`?

Reply:
This function can only be used from the useMatomo Hook, which used the MatomoTracker instance, that won't resolve without the window object. So guess this check wouldn't be needed